package.json

```diff
@@ -34,7 +34,7 @@
 		"storybook:build": "build-storybook -c .storybook -o .storybook-static",
 		"storybook": "start-storybook -p 8888",
 		"test": "jest",
-		"version": "yarn genGlobalChangeLog && prettier --write -- \"**/CHANGELOG.md\" && git add -- \"CHANGELOG.md\" \"packages/clay-css/src/scss/_license-text.scss\""
+		"version": "prettier --write -- \"**/CHANGELOG.md\" && git add -- \"CHANGELOG.md\" \"packages/clay-css/src/scss/_license-text.scss\""
```

Comment:
Out of curiosity, do we still need the `prettier` call here? Before you added the global changelog script this line used to look [like this](https://github.com/liferay/clay/blame/d7e00467ac6e72353ffd69d7841fcfbf4bcfb86a/package.json#L36):

```
		"version": "yarn run format:all"
```

Is the current version better because it's faster?

Reply:
@wincent yes this version is is more specific for what we actually want and it is faster. The changelog genreation that happens via lerna doesn't follow prettier formatting so we have to make sure and run that before we have the "publish" commit